common/views/components/ImageGalleryV2/ImageGalleryV2.js

```diff
@@ -136,34 +136,45 @@ class ImageGallery extends Component<Props, State> {
                   )}
 
                   <div
-                    style={{
-                      position: 'sticky',
-                      top: '10px',
-                      paddingRight: '10px',
-                      paddingBottom: '10px',
-                      display: 'flex',
-                      justifyContent: 'flex-end',
-                      visibility: isActive ? 'visible' : 'hidden',
-                      zIndex: 3,
-                    }}
+                    className={classNames({
+                      [spacing({ s: 3 }, { padding: ['top'] })]: true,
+                      'image-gallery-v2__close absolute': true,
+                    })}
                   >
-                    <Control
-                      url={`#gallery-${id}`}
-                      type={`dark`}
-                      text={`close`}
-                      icon={`cross`}
-                      clickHandler={event => {
-                        trackEvent({
-                          category: `Control`,
-                          action: 'close ImageGallery',
-                          label: this.props.id,
-                        });
-
-                        setTimeout(() => {
-                          this.setState({ isActive: false });
-                        }, 0);
+                    <div
+                      className={classNames({
+                        'flex flex-end': true,
+                        [spacing(
+                          { s: 3 },
+                          { padding: ['right', 'bottom'] }
+                        )]: true,
+                      })}
+                      style={{
+                        position: 'sticky',
+                        top: '18px',
+                        transform: `translateX(calc((100vw - 100%) / 2))`,
+                        visibility: isActive ? 'visible' : 'hidden',
+                        zIndex: 3,
                       }}
-                    />
+                    >
+                      <Control
+                        url={`#gallery-${id}`}
+                        type={`light`}
+                        text={`close`}
+                        icon={`cross`}
+                        clickHandler={event => {
+                          trackEvent({
+                            category: `Control`,
+                            action: 'close ImageGallery',
+                            label: this.props.id,
+                          });
+
+                          setTimeout(() => {
```

Comment:
what's this little ditty about?

Reply:
Good question – whatever it was about originally doesn't appear to be necessary any more. Will remove.